tests/test_settings.py

```diff
@@ -24,6 +25,11 @@ def test_settings_from_default_location(settings_path: Path) -> None:
 
     assert settings.defaults == {"foo": "bar"}
 
+@pytest.mark.skipif(OS != 'windows')
```

Comment:
Could you please use `platform.system()` do identify the OS instead of `copier.tools.OS`? We use the latter for internal OS identification, which may be [patched for testing](https://github.com/copier-org/copier/blob/f842729d998961bf10a88d36940b75cc4641d150/tests/test_tasks.py#L164).

```suggestion
@pytest.mark.skipif(platform.system() != "Windows")
```